subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/pmd/PmdPluginVersionIntegrationTest.groovy

```diff
@@ -258,6 +258,66 @@ class PmdPluginVersionIntegrationTest extends AbstractPmdPluginVersionIntegratio
         output.contains "\tEnsure you override both equals() and hashCode()"
     }
 
+    void "can configure number of threads on good code"() {
```

Comment:
These added tests are 👍 .
One more thing that could be worth testing, if possible, is that PMD is effectively using multiple threads. One way of doing that could be checking logs if a debug output from PDM provides that information.

Reply:
An alternative would be to see if `PmdInvoker` can be tested to confirm that the new option is passed down.

Reply:
The pull request is currently blocked on this issue.

Reply:
Sorry, looks like I missed your previous answer.
As I said, verifying that would have been great if possible. But that does not look easily possible.
I've pushed the PR through CI to see if there are any issue.